worker/worker/request_handlers.go

```diff
@@ -486,6 +496,16 @@ func (s StreamContext) getTranscodingFileName() string {
 		s.getStreamName())
 }
 
+func (s StreamContext) getAudioTranscodingFileName() string {
+	return fmt.Sprintf("%s/%d/%s/%s/%s/%s.m4a",
+		cfg.StorageDir,
+		s.teachingYear,
+		s.teachingTerm,
+		s.courseSlug,
+		s.startTime.Format("2006-01-02_15-04"),
```

Comment:
This probably introduces the bug describes in #882 also to audio files. Should we change this to 
```golang
return fmt.Sprintf("%s/%d/%s/%s/%d/%s.m4a",
 		cfg.StorageDir,
 		s.teachingYear,
 		s.teachingTerm,
 		s.courseSlug,
 		s.streamId, 
                s.streamName)
``` 
similar to the large thumbnails for the new start page?

Reply:
Changing the filename to streamID-Version.mp4/m4a seems like a good idea